pyproject.toml

```diff
@@ -73,81 +73,81 @@ docs = ["recommonmark", "sphinx", "sphinx-click", "sphinx-rtd-theme", "sphinxcon
+  {signature = "948237ebd4761233e14aa6ff0cf8acc9920f0132b270e2af987e7957e6b31f00"},
+  {signature = "1f08f266f7eb6932cdfcd6ba256067552dec3d85f9d7599612e94360e9890ec1"},
+  {signature = "0cdf53e226fcbdc95a5f9533de3b79e0d6993a411ebf5976bdacf416d1845e95"},
+  {signature = "29c93895d1b1703d8a6b9b0210bb2b93dbd210273cf9ee210e740e109533e977"},
+  {signature = "c7cf2a9e938b89a0fa8850fda1110dd7a1f9263648389fec5b8dbacd9e06c918"},
+  {signature = "a47a9cc3ceac2df681683678569cde5e5298930090d26665be80066a569378a5"},
+  {signature = "24db34759e7eee9168990583b3551aa6c019176e75a1655bf94e07c62e26cc19"},
+  {signature = "ba554d07d9d78f56f9a0e670e094d0247b3d51fb95c4112e0a9317d4cc960855"},
+  {signature = "baa9ed3d871b6392cfc79a17fcffa562d2b9aa51cb548c5939c2a41f1acc8467"},
+  {signature = "27eb03ed9af3707094c142b6bd485793e579ae1c59ce13193cb6b9f31c0f011f"},
+  {signature = "2a3cb329b81351e357b09f1b97323ff726e72bd5ff8427c9295e6ef68226e1d1"},
+  {signature = "586f9df1870fcfafd3070dc553e97db25b3fce82c9144602567d69a86597c06f"},
+  {signature = "842533f44cf32fb3d93a7f56227977aa4f16caafe82ace8f5f4de27d750c1ec1"},
+  {signature = "d039c652f27c4d42026c5b5c9be31bfe368b283b24248e98d92f131272580053"},
 
   # test secrets
-  "6c70b4a712ada69f2a6c919edc783cf485b6637e3c5429de376ef1a451a042e4", # tests/data/scan_folder/donotscan.txt
-  "357841766ca69979608e4e7fb1da2addeafe6d15cade15466c39d903cc0488a8", # tests/data/scan_folder/test.txt
-  "743b893e30777d9c4e28d3d341ca4ba1c2541a9c62b497dece75c2a64420e770", # tests/test_folder_scanner.py
-  "358a1ae040ab71b3aafeb4318e5d9e318ff1b681c5bf0e99d381584a33ee7833", # tests/data/scan_folder/scan_sub_folder/sub_folder_test.txt
-  "9d1b278fa384c9dc58d607130d97740910309d13b29f13f7ed6348738ea4f32c", # tests/test_folder_scanner.py 16-Nov-2021
-  "9f675d4d7f43e484e07fb199a1e7ec372ef204631c1f4ec054acc3119fbde4cf", # build_and_publish_docker_file 13-Jun-2019
-  "88152f1d10077f417ee5aea5fa86adaa5b74d3f2de2e87ce4b4aa5cd79c7b2f5", # nothing file, 30-Dec-2016
+  {signature = "6c70b4a712ada69f2a6c919edc783cf485b6637e3c5429de376ef1a451a042e4", reason = "tests/data/scan_folder/donotscan.txt"},
+  {signature = "357841766ca69979608e4e7fb1da2addeafe6d15cade15466c39d903cc0488a8", reason = "tests/data/scan_folder/test.txt"},
+  {signature = "743b893e30777d9c4e28d3d341ca4ba1c2541a9c62b497dece75c2a64420e770", reason = "tests/test_folder_scanner.py"},
+  {signature = "358a1ae040ab71b3aafeb4318e5d9e318ff1b681c5bf0e99d381584a33ee7833", reason = "tests/data/scan_folder/scan_sub_folder/sub_folder_test.txt"},
+  {signature = "9d1b278fa384c9dc58d607130d97740910309d13b29f13f7ed6348738ea4f32c", reason = "tests/test_folder_scanner.py 16-Nov-2021"},
+  {signature = "9f675d4d7f43e484e07fb199a1e7ec372ef204631c1f4ec054acc3119fbde4cf", reason = "build_and_publish_docker_file 13-Jun-2019"},
+  {signature = "88152f1d10077f417ee5aea5fa86adaa5b74d3f2de2e87ce4b4aa5cd79c7b2f5", reason = "nothing file, 30-Dec-2016"},
```

Comment:
We should review these at some point for what's still relevant, given the path pattern and entropy exclusions... but, you know, problem for future us.